### PR TITLE
feat(layout-logs): add debug view

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-663)

- Add debug checkbox to display Nginx logs

Capture d’écran 2023-03-07 à 15 09 05

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(layout-logs): add debug view - 2023-03-07T14:09:22Z](https://new.console.qovery.com/organization/141c07c8-0dd9-4623-983b-3fdd61867255/project/977bd069-20f7-4b24-bd95-28b44174cf43/environment/22bdc14d-b2ff-4b58-8f9d-eacd5a058338/services/general)

 Another comment will be posted when deployments are terminated

File: libs/pages/logs/application/src/lib/page-application-logs.tsx [hunk 1]
```diff
@@ -72,6 +82,19 @@ export function PageApplicationLogs() {
     },
   })
 
+  useWebSocket(nginxLogsUrl, {
+    onMessage: (message) => {
+      const data = { ...JSON.parse(message?.data), pod_name: 'nginx' }
```
Comment (bdebon) [inline libs/pages/logs/application/src/lib/page-application-logs.tsx:87]: smart

**Review by bdebon — APPROVED** [review]
Works well and smart way of implementing that!

**Pull request merged**